sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -582,13 +618,42 @@ export interface TopicRuntimeProperties {
     subscriptionCount?: number;
 }
 
+// @public
+export type TransferProgressEvent = {
+    loadedBytes: number;
+};
+
 // @public
 export interface TryAddOptions {
     // @deprecated (undocumented)
     parentSpan?: Span | SpanContext | null;
     tracingOptions?: OperationTracingOptions;
 }
 
+// @public
+export interface WebResourceLike {
```

Comment:
out of curiosity - if we ever do a major version bump for this package, would we remove all this ported surface from core-http?

Reply:
I think so.  If we do major version bump, we could either remove `_response` and provide the `onResponse` callback, or we keep the property and just return the PipelineResponse, so ported types and transformations like below are not needed 
```ts
export function toHttpResponse(response: PipelineResponse): HttpResponse {
  return {
    request: toWebResourceLike(response.request),
    status: response.status,
    headers: toHttpHeaderLike(response.headers),
  };
}
```